src/utils/config.ts

```diff
@@ -27,7 +27,7 @@ const configParsers = {
 				'Please set your OpenAI API key via `aicommits config set OPENAI_KEY=<your token>`'
 			);
 		}
-		parseAssert('OPENAI_KEY', key.startsWith('sk-'), 'Must start with "sk-"');
+		parseAssert('OPENAI_KEY', /^((sk-|no_api_key){1}[a-zA-Z0-9]*)$/.test(key), 'Must start with "sk-" or "no_api_key"');
```

Comment:
can we completely remove this check? My OpenAI instance requires an API key which doesn't start by `sk-`, but still requires an API key.

Reply:
I can remove it in a separate commit.